setup.py

```diff
@@ -2,19 +2,19 @@
 import sys
 from setuptools import setup
 
-requires = ['awscli>=1.11.0']
+requires = []
 
 setup(
-    name='awscli-plugin-s3-proxy',
-    packages=['awscli_plugin_s3_proxy'],
+    name='awscli-plugin-proxy',
+    packages=['awscli_plugin_proxy'],
     version='0.5',
```

Comment:
As we are removing the dependency on awscli, it makes sense to me to have a different version number.